scss/lessonCard.scss

```diff
@@ -0,0 +1,21 @@
+.card-title {
+  font-size: 1.5rem;
+  font-weight: bold;
+  line-height: 1.33;
+}
+.icon-messages {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 75%;
+  justify-content: space-between;
+  white-space: nowrap;
+}
+.icon-text {
+  font-size: 0.8125rem;
+  line-height: 1.38;
+}
+.lesson-description {
+  font-size: 0.9375rem;
+  line-height: 1.6;
+  margin-top: 0.7rem;
```

Comment:
could you give it a `mt-x` instead of using `margin-top`?

Reply:
Changed to mt-2 👍